etc/profile-a-l/google-chrome-beta.profile

```diff
@@ -8,10 +8,16 @@ include globals.local
 noblacklist ${HOME}/.cache/google-chrome-beta
 noblacklist ${HOME}/.config/google-chrome-beta
 
+noblacklist ${HOME}/.config/chrome-beta-flags.conf
+noblacklist ${HOME}/.config/chrome-beta-flags.config
+
 mkdir ${HOME}/.cache/google-chrome-beta
 mkdir ${HOME}/.config/google-chrome-beta
 whitelist ${HOME}/.cache/google-chrome-beta
 whitelist ${HOME}/.config/google-chrome-beta
 
+whitelist ${HOME}/.config/chrome-beta-flags.conf
```

Comment:
Same remarks as above.

Reply:
I've tested the profiles as they are now and google-chrome-beta does seem to look for `chrome-beta-flags.conf / chrome-beta-flags.config`.